en-US/Conductors_Companion/03_Preparing_Quests_and_Adventures/Conductor_Prep_in_Eight_Steps/6_Outline_Important_NPCs.md

```diff
@@ -0,0 +1,121 @@
+To really bring the NPC to life, you can then model their personality and roleplaying off characters from your favorite books, TV shows, or movies, switching up gender and other traits to make them feel fresh.
+
+<!-- style:sidebar -->
+> **Sample Names** \
+> There are many ways to name your NPCs.
+> For example, you can take names of people you know and spelling them backwards.
+> Alternatively, you can use an online name generator, a book of names, or one of the following names:
+> <!-- spell-checker:disable -->
+>
+> |            |       |          |               |        |         |           |
+> |:-----------|:------|:---------|:--------------|:-------|:--------|:----------|
+> | Hollyfang  | Ryan  | Cunovin  | Emeraldstorm  | Sanche | Pulmia  | Samantha  |
+> | Lawknocker | Poine | Aegipan  | Sharpwhisker  | Taran  | Nireus  | Icebrood  |
+> | Anvilcloud | Sabra | Parnell  | Rainsoother   | Lowell | Clifton | Lionfall  |
+> | Somerhild  | Halia | Leofwen  | Earthdancer   | Boyle  | Margry  | Bellinda  |
+> | Eupalamos  | Cared | Godebert | Wynefreede    | Peter  | Derkos  | Savannah  |
+> | Heromaker  | Toril | Osborne  | Goosewalker   | Arnald | Claudia | Lapithes  |
+> | Dragonknee | Paul  | Redspur  | Shadowblood   | Neale  | Timothy | Halisera  |
+> | Macebound  | Hilda | Denston  | Brightwhisper | Hailey | Bayard  | Goldbane  |
+> | Moongazer  | Drew  | Caroline | Shieldrazor   | Vesta  | Turstin | Fireheart |
+> | Hildegard  | Jeger | Goddard  | Glasscleaver  | Roger  | Judithe | Mudteeth  |
+> | Wyrmriver  | Maya  | Beneger  | Anthonette    | Symond | Radolf  | Nicholas  |
+> | Guinevere  | Dauid | Chulisa  | Swiftcaller   | Meryll | Avelin  | Reothine  |
+> | Lightblade | Chase | Jocelyn  | Goblinchaser  | Kaylee | Hudson  | Halstein  |
+> | Catfinger  | Sadie | Sybaris  | Angelbright   | Josiah | Latisha | Thornhelm |
+> | Ebondazer  | Tansa | Rycheld  | Spiderhunter  | Franny | Stella  | Randwulf  |
+> | Flamestar  | Jayce | Cullive  | Cloudlover    | Albin  | Gerland | Tantalos  |
+> | Nicholina  | Bryde | Madison  | Swordbuckle   | Edmund | Sysley  | Johannes  |
+> | Graywillow | Aisa  | Annabel  | Bronzestone   | Sydnee | Lausus  | Isemeine  |
+> | Thelexion  | Sulen | Teukros  | Monsterthumb  | Colton | Florens | Millhand  |
+> | Whitetail  | Levi  | Maronne  | Doombrissle   | Raffe  | Sighard | Sebastian |
+
+<!-- spell-checker:enable -->
+
+___
+Use the following table to choose a random ancestry for your NPC.
+
+| 1d10 | Ancestry | \| | 1d10 | Ancestry                         |
+|:----:|:---------|:--:|:----:|:---------------------------------|
+|   1  | Human    | \| |   6  | Mixed (Roll twice on this table) |
+|   2  | Elf      | \| |   7  | Gnome                            |
+|   3  | Dwarf    | \| |   8  | Orc                              |
+|   4  | Halfling | \| |   9  | Dragonborn                       |
+|   5  | Goblin   | \| |  10  | Tiefling                         |
```

Comment:
This table uses the width of the page more effectively than it would if it were just 10 rows. I did the same with the following tables.

Whether or not we include that middle column with the pipe symbols is certainly something to think about though.

<img width="495" height="240" alt="The current ancestry table in context." src="https://github.com/user-attachments/assets/88329efb-110f-41b7-9c62-6786ec510cd4" />

Reply:
Certainly worth a try! I'll test it and post a screenshot.